runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkExecutionEnvironments.java

```diff
@@ -164,6 +164,11 @@ public static StreamExecutionEnvironment createStreamExecutionEnvironment(
                     ? ExternalizedCheckpointCleanup.RETAIN_ON_CANCELLATION
                     : ExternalizedCheckpointCleanup.DELETE_ON_CANCELLATION);
       }
+
+      long minPauseBetweenCheckpoints = options.getMinPauseBetweenCheckpoints();
+      flinkStreamEnv
```

Comment:
With this, we might always be overriding the Flink default if it changes, or if it becomes configurable in the Flink config in the future.

I think we should check whether the value is different from the "Beam default" and only set it then.